rizwan171/trello-clone
# Tc 45 jest tests part 1

This is part 1 of the jest changes. Instead of dropping hundreds of files into PR at once when I finish all the tests, I thought to split it out into multiple smaller PRs to make it easier to review.

How to check if the tests work:
- Run `yarn install` to install all the new dependencies
- Run `yarn test` and confirm everything passes
